test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -1341,7 +1339,8 @@ func TestUnReservePreBindPlugins(t *testing.T) {
 			}
 
 			if test.fail {
```

Comment:
should it be `test.wantUnschedulable` instead?

Reply:
Change to `test.reject` which corresponding to `PreBindPlugin.rejectPreBind`.